build/Dockerfile.s390x

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2019-2022 Tigera, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG QEMU_IMAGE=calico/go-build:v0.84-s390x
+FROM ${QEMU_IMAGE} as qemu
+
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.7 AS ubi
+
+# Enable non-native builds of this image on an amd64 hosts.
+# This must be the first RUN command in this file!
+COPY --from=qemu /usr/bin/qemu-*-static /usr/bin/
+
+# Update base packages to get security updates.
+RUN microdnf update
+
+# Add in top-level license file
+RUN mkdir /licenses
+COPY LICENSE /licenses
+
+FROM scratch
+COPY --from=qemu /usr/bin/qemu-*-static /usr/bin/
```

Comment:
I see that this line exists in the arm64 image as well, but I'm curious if we need this in the final image or if it's just compile time?

Reply:
I think we need copy `qemu-*-static` to scratch to identify the image `arch`.